robot-core/src/main/java/org/obolibrary/robot/Template.java

```diff
@@ -653,7 +643,13 @@ private void addLabels() {
           entity = dataFactory.getOWLEntity(EntityType.NAMED_INDIVIDUAL, iri);
           break;
       }
-      checker.add(entity, label);
+
+      if (id != null) {
+        checker.add(entity, id);
+      }
+      if (label != null) {
+        checker.add(entity, label);
```

Comment:
This is probably the only thing someone need to physically check: does adding both (id AND label) ensure that both syntaxes can interpreted by the entitychecker? I assume so!

Reply:
Yes, this should be tested, but I feel pretty confident about it. The bigger problem I've noticed is that `QuotedEntityChecker.add(entity, name)` will happily, quietly replace existing names.